test/common/git-utils.sh

```diff
@@ -13,6 +13,12 @@ set -eu
 unset GIT_DIR GIT_EXEC_PATH GIT_PREFIX GIT_REFLOG_ACTION GIT_WORK_TREE
 
 GITHUB_BASE="${GITHUB_BASE:-cockpit-project/cockpit}"
+
+# Override GITHUB_BASE if working with the 'bots' repository
+if [ "${GITHUB_REPO}" = "bots" ]; then
+    GITHUB_BASE="cockpit-project/bots"
+fi
+
```

Comment:
To clarify: I assume this is *not* related to refreshing node_modules, as the PR description insinuates? Anaconda's `GITHUB_BASE=rhinstaller/anaconda-webui` should make this resolve to `GITHUB_REPOSITORY=rhinstaller/node-cache`.

I figure this breaks `test/common/make-bots` instead? It would IMHO make much more sense to put `GITHUB_BASE=cockpit-project/bots` into test/common/make-bots , that would be a more appropriate place for that special case.

Note that the project name doesn't matter, this is just for the org.

Reply:
yes it's `make-bots`, I am setting our our repository GITHUB_BASE because of the node-cache, and that causes the helper tools to try to fetch bots from our org in github.